### DISCUSSION TITLE
Using Sequelize (ORM) within Cypress tests

### DISCUSSION THREAD

**Discussion by mailtwogopal** [cypress-io/cypress#27115] [Questions and Help]
Hi,
I am trying to implement Sequelize on my cypress tests for a complex app, to interact with database with queries and to verify expected results. I followed almost all the from official documentation or from popular searches but none of them help get me connected to db. We are using mssql and we end up getting webpack error even though dialect options are provided correctly. 

Link to Sequelize is [here](https://sequelize.org/)

Is there a reference on any of the users of Cypress tried with any ORM npm packages? Were there any successful implementations yet? Any help on this would be greatly appreciated. 

Thanks.

**Comment by bahmutov** [comment]
I have a lesson about using Postgres with Sequalize from Cypress specs in my course https://cypress.tips/courses/cypress-plugins
